### DISCUSSION TITLE
With Internet explorer going out support, how do we switch to Edge in Eclipse ?

### DISCUSSION THREAD

**Discussion by niraj-modi** [eclipse-platform/eclipse.platform.swt#209] [Q&A]
Internet explorer is going out support:
https://support.microsoft.com/en-us/windows/internet-explorer-downloads-d49e1f0d-571c-9a7b-d97e-be248806ca70

Microsoft says switch to Edge, but current Edge implementation has below 2 blocking limitations w.r.t. usage in Eclipse:
- #212 
- #213 

@nnemkin Any idea what is the way out here ?

Top level bug for improving Edge support:
[Bug 575660](https://bugs.eclipse.org/bugs/show_bug.cgi?id=575660) - [Browser][Edge] Improve support for embedding Edge on Windows

Let's use this discussion thread to explore possible options.

**Comment by merks** [comment]
It also seems to me that users need to install some additional thing before Edge even works, so that always struck up as not an out-of-the-box replacement...

**Reply by niraj-modi** [reply]
Edge's WebView2 Runtime is needed to be installed on the machine for embedded edge to work, for more details please see below SWT FAQ:
https://www.eclipse.org/swt/faq.php#howuseedge

**Comment by nnemkin** [comment]
Only the user-facing application is going out of support. The component itself is there to stay for a long, long time.

**Reply by niraj-modi** [reply]
> Only the user-facing application is going out of support. The component itself is there to stay for a long, long time.

Understand, but since new web-site are hardly(or less) verified/tested on IE so it's less likely user will want to continue to use embedded IE. So, gradually improving Edge implementation is the right approach and hence getting Edge implementation more close to IE from Eclipse point of view makes much more sense.

@nnemkin 
Please have a look on below discussion w.r.t. to extra processes of embedded Edge:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=571278#c9 - bug now moved to #212
